doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2003,10 +2003,11 @@ applied to an unknown structure instance (an implicit argument) and a
 value. The complete documentation of canonical structures can be found
 in :ref:`canonicalstructures`; here only a simple example is given.
 
-.. cmd:: Canonical {? Structure } @qualid
+.. cmd:: {? #[local] } Canonical {? Structure } @qualid
 
    This command declares :token:`qualid` as a canonical instance of a
-   structure (a record).
+   structure (a record). When the :g:`#[local]` attribute is given the effect
```

Comment:
```suggestion
   structure (a record). When the :g:`#[local]` attribute is given, the effect
```

Reply:
Shouldn't we use Local instead of #[local]?

Reply:
`Local` is also supported but since we are leaning toward replacing this kind of modifier with attributes in the long-run, I think it makes sense to start nudging toward the use of the latter.